.github/workflows/build-dependencies.txt

```diff
@@ -27,23 +20,11 @@ libselinux-dev
 libssl-dev
 libtool
 libudev-dev
-lsscsi
-mdadm
-nfs-kernel-server
-pamtester
-parted
+linux-headers-generic
```

Comment:
Without dkms we don't need headers too, AFAIK

Reply:
I will try this.